src/classes/application.js

```diff
@@ -12,22 +13,35 @@ module.exports = class Application extends Base {
     const config = require(path.join(process.cwd(), 'config/environment.js'))(process.env.NODE_ENV)
     const MiddlewareRouter = require(path.join(process.cwd(), 'app/middleware.js'))
     const Router = require(path.join(process.cwd(), 'app/router.js'))
+    const log = new Log()
 
+    log.trace('Ash server creating express app instance')
     const app = express()
     _app.set(this, app)
 
-    const log = new Log()
-
-    log.info('Ash server loading middleware')
+    log.trace('Ash server loading middleware')
     const middlewareDir = path.join(process.cwd(), 'app/middleware')
     loadMiddleware(MiddlewareRouter.definition, app, middlewareDir)
 
-    log.info('Ash server loading routes')
+    log.trace('Ash server loading routes')
     const options = {routesDir: path.join(process.cwd(), 'app/routes')}
     app.use(createRoutes(Router.definition, options))
 
+    const initializerDir = path.join(process.cwd(), 'app/initializers')
```

Comment:
`process.cwd()` returns the current working directory of the shell that runs the command. Is that desirable behavior, or would it be best to pin the "working directory" to the top-level of the Ash project itself.

Also note that by using `process.cwd()` in determining the location of the initializers directory, it will prevent you from loading any initializers defined by the Ash framework itself. We have yet to see if that is something we want to do, however.